vhdbuilder/packer/components.json

```diff
@@ -351,9 +349,7 @@
       "amd64OnlyVersions": [
         "1.4"
       ],
-      "multiArchVersions": [
-        "v1.11.9"
```

Comment:
defaultbackend is not used on 1.21.0+ nodes anymore, so 1.22.0+ arm64 nodes don't need it neither.